src/test/java/com/epam/tc/hw2/ex1/LoginPageTest.java

```diff
@@ -0,0 +1,120 @@
+            + "EPAM project");
+        WebElement textUnder2 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To be flexible')]"));
+        textUnder2.isDisplayed();
+        softly.assertThat(textUnder2.getText()).isEqualTo("To be flexible and\n"
+            + "customizable");
+        WebElement textUnder3 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To be multiplatform')]"));
+        textUnder3.isDisplayed();
+        softly.assertThat(textUnder3.getText()).isEqualTo("To be multiplatform");
+        WebElement textUnder4 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'Already have good base')]"));
+        textUnder4.isDisplayed();
+        softly.assertThat(textUnder4.getText()).isEqualTo("Already have good base\n"
+            + "(about 20 internal and\n"
+            + "some external projects),\n"
+            + "wish to get more…");
+        //Assert that there is the iframe with “Frame Button” exist
+        WebElement frameWithButton = webDriver.findElement(By.id("frame"));
+        frameWithButton.isDisplayed();
+        //Switch to the iframe and check that there is “Frame Button” in the iframe
+        webDriver.switchTo().frame(frameWithButton);
+        WebElement frameButton = webDriver.findElement(By.id("frame-button"));
+        frameButton.isDisplayed();
+        //Switch to original window back
+        webDriver.switchTo().defaultContent();
+        //Assert that there are 5 items in the Left Section are displayed and they have proper text
+        WebElement homeOnLeftSection =
+            webDriver.findElement(By.xpath("//*[@class='sidebar-menu left']//a[@href='index.html']"));
+        homeOnLeftSection.isDisplayed();
+        softly.assertThat(homeOnLeftSection.getText()).isEqualTo("Home");
+        WebElement contactOnLeftSection =
+            webDriver.findElement(By.xpath("//*[@class='sidebar-menu left']//a[@href='contacts.html']"));
+        contactOnLeftSection.isDisplayed();
+        softly.assertThat(contactOnLeftSection.getText()).isEqualTo("Contact form");
+        WebElement serviceOnLeftSection = webDriver.findElement(By.xpath("//span[contains(text(),'Service')]"));
+        serviceOnLeftSection.isDisplayed();
+        softly.assertThat(serviceOnLeftSection.getText()).isEqualTo("Service");
+        WebElement metalsAndColorsOnLeftSection =
+            webDriver.findElement(By.xpath("//*[@class='sidebar-menu left']//a[@href='metals-colors.html']"));
+        serviceOnLeftSection.isDisplayed();
+        softly.assertThat(metalsAndColorsOnLeftSection.getText()).isEqualTo("Metals & Colors");
+        WebElement elementPacksOnLeftSection = webDriver.findElement(By.xpath("//span[contains(text(),'Element')]"));
+        elementPacksOnLeftSection.isDisplayed();
+        softly.assertThat(elementPacksOnLeftSection.getText()).isEqualTo("Elements packs");
```

Comment:
All web elements for the menu could be getter as `List<WebElement>` and assertion could be used through collection

Reply:
looks like this comment was ignored